src/sdk/namespace_fs.js

```diff
@@ -1426,6 +1426,15 @@ class NamespaceFS {
         }
     }
 
+    _is_force_md5_enabled(object_sdk) {
```

Comment:
If you can please add JSDoc to the function (we try to add it in new functions that we add).